bot/exts/evergreen/trivia_quiz.py

```diff
@@ -226,18 +481,20 @@ async def leaderboard(self, ctx: commands.Context) -> None:
 
     @staticmethod
     async def send_score(channel: discord.TextChannel, player_data: dict) -> None:
-        """A function which sends the score."""
+        """Send the current scores of players in the game channel."""
         if len(player_data) == 0:
             await channel.send("No one has made it onto the leaderboard yet.")
             return
 
-        embed = discord.Embed(colour=discord.Colour.blue())
-        embed.title = "Score Board"
-        embed.description = ""
+        embed = discord.Embed(
+            colour=Colours.blue,
+            title="Score Board",
+            description="",
+        )
 
         sorted_dict = sorted(player_data.items(), key=lambda a: a[1], reverse=True)
```

Comment:
You already imported `operator`, maybe use `itemgetter(1)`? :-)

Reply:
yessssssssssss lambdas bad! imported functions good!